build.yaml

```diff
@@ -16,8 +16,26 @@ targets:
     sources:
       - tool/builder.dart
       - tool/mustachio/builder.dart
+  testing:
```

Comment:
Does this change how the builder is invoked to generate foo.dart?

Reply:
No, `pub run build_runner build` still works fine.